front_end/src/index.js

```diff
@@ -1,8 +1,49 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import registerServiceWorker from "./registerServiceWorker";
+
+// Material-UI Imports Below
+import { MuiThemeProvider, createMuiTheme } from "@material-ui/core";
+import { red } from "@material-ui/core/colors";
+
+const theme = createMuiTheme({
+  typography: {
+    fontFamily: [
+      "-apple-system",
+      "BlinkMacSystemFont",
+      '"Segoe UI"',
+      "Roboto",
+      '"Helvetica Neue"',
+      "Arial",
+      "sans-serif",
+      '"Apple Color Emoji"',
+      '"Segoe UI Emoji"',
+      '"Segoe UI Symbol"'
+    ].join(",")
+  },
+  palette: {
+    primary: {
```

Comment:
and those are the color palettes from material UI you talked about?

Reply:
Yeah. It applies the theme to all Material UI components.